index.html

```diff
@@ -77,33 +78,33 @@ <h5 class='offcanvas-title' id='offcanvasExampleLabel'>Menu</h5>
         <div class='offcanvas-body'>
           <ul class="offcanvas-menu  " >
             <li class="" >
-              <a class="btn btn-secundary" href="">
-                  <button class="btn btn-primary">About</button>     
+              <a class="" type="button" href="./views/about.html">
+                  <button class="btn btn-primary" type="button">About</button>     
               </a>                         
             </li>            
             <li class="" >
-              <a class="btn btn-secundary" href="">
-                  <button class="btn btn-primary">Program</button>     
+              <a class="" type="button" href="">
+                  <button class="btn btn-primary" type="button">Program</button>     
```

Comment:
- [ ] Please It should be possible to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too I see that you added a link to the about page but it would be better to add another one to which send the user to the home page other than the one in the logo.
![image](https://user-images.githubusercontent.com/74744787/226133563-a1b06366-b622-46ec-84b3-2c0e03e72d04.png)

Reply:
done